src/libraries/System.Drawing.Common/tests/StringFormatTests.cs

```diff
@@ -84,7 +84,7 @@ public void Ctor_Format()
             }
         }
 
-        [Fact]
+        [ConditionalFact(Helpers.IsDrawingSupported)]
```

Comment:
So this indeed fixes: #35917 ?

Reply:
Nevermind, just saw we're skipping the hole assembly.

Reply:
Well, that one ;-).  There are too many.